### DISCUSSION TITLE
Certificate

### DISCUSSION THREAD

**Discussion by main22** [community/community#67242] [Code Security]
### Select Topic Area

Question

### Body

may i know platform for certificate this topic

**Comment by pnjrmdn** [comment]
Skilvul and Cisco better for it

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
